addons/common/XEH_postInit.sqf

```diff
@@ -263,6 +263,35 @@ if (_currentVersion != _previousVersion) then {
 
 call FUNC(checkFiles);
 
+// Let the server know that we have ACE loaded
+missionNamespace setVariable [QGVAR(aceLoaded_) + getPlayerUID player, true, 2];
+
+// This handles clients joining that do not have ACE loaded
+if (isServer) then {
+    addMissionEventHandler ["PlayerConnected", {
+        [{
+            params ["", "_uid", "", "", "_owner"];
+
+            // If ACE is not loaded, kick unit
+            if !(missionNamespace getVariable [QGVAR(aceLoaded_) + _uid, false]) then {
```

Comment:
Either variable broadcast is not being done on time or something's wrong with this check. Getting kicked off dedicated MP hosted off my own machine.

Reply:
Increasing timeout on this WAE to 10 seconds fixed it, but I don't think a magic number exists and we're subject to player/server perfomance and network connection here.

Reply:
First load, timeout at 9 seconds: Kicked for no/outdated ACE.
Second load, same timeout: got in fine.

This isn't a reliable implementation.